script/buildSQLGeom.py

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/env python
+
+"""
+This module will accept an input text file with (x,y,[z]) coordinates in a CSV-format list
+and generate a SQL string that will add these coordinates as a PostGIS Polygon
+to sites.geometry within a BETYdb instance.
+
+A header row should be included in the input file, although the column names do not matter. If a z column
+for altitude is not included, the script will attempt to get it from USGS NED (US) or Google (global).
```

Comment:
I have a key.. But to confirm, points in US query USGS by default, correct?
On Thu, Nov 19, 2015 at 4:23 PM Max Burnette <notifications@github.com>
wrote:

> In script/buildSQLGeom.py
> <https://github.com/PecanProject/bety/pull/369#discussion_r45410118>:
>
> > @@ -0,0 +1,236 @@
> > +#!/usr/bin/env python
> > +
> > +"""
> > +This module will accept an input text file with (x,y,[z]) coordinates in a CSV-format list
> > +and generate a SQL string that will add these coordinates as a PostGIS Polygon
> > +to sites.geometry within a BETYdb instance.
> > +
> > +A header row should be included in the input file, although the column names do not matter. If a z column
> > +for altitude is not included, the script will attempt to get it from USGS NED (US) or Google (global).
>
> I just committed a version with the google elevation API. However, I am
> not using it by default - it requires the user to point to a text file with
> a valid Google Maps API key (they use for traffic quotas).
>
> I can provide you with my API key if you like, or you can generate your
> own:
> https://developers.google.com/maps/documentation/elevation/get-api-key
>
> If that file is present and *line 130* points to it, the API should work
> and we can call it instead of the USGS API on *line 233*.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PecanProject/bety/pull/369/files#r45410118>.
>